test/plugins/features/movies/controller.test.js

```diff
@@ -23,5 +23,128 @@ describe('movie controller', () => {
     });
 
   });
-  
+
+
+  describe('retrieve a list of movies', () => {
+
+    it('returns all movies', () => {
+      return new Movie().fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((count) => {
+        return [Controller.display(), count];
```

Comment:
Nit: Its not always necessary to pass dependent data by resolving it across the promise chain. For this example its not too complex, but imagine a scenario where a `.then` handler needed three or four values from earlier calls. Using an array to pass these values would be harder to read and more fragile to refactor.

One common pattern is to initialize a variable outside of the scope of the promise chain and set and reference its value from within the promise chain. For example:
```js
let data;
return new Movie().fetchAll()
.then((movies) => {
   // set data
})
.then(() => {
  // retrieve data
});
```